docker-compose.yml

```
@@ -0,0 +1,60 @@
+services:
+  prometheus:
+    image: prom/prometheus
+    container_name: prometheus
+    command:
+      - '--config.file=/etc/prometheus/prometheus.yml'
+    ports:
+      - "9090:9090"
+    restart: unless-stopped
+    volumes:
+      - ./docker/prometheus/prometheus:/etc/prometheus.yml
+      - prom_data:/prometheus
+    networks:
+      - metrics
+
+  grafana:
+    image: grafana/grafana
+    container_name: grafana
+    ports:
+      - "3000:3000"
+    restart: unless-stopped
+    environment:
+      - GF_SECURITY_ADMIN_USER=admin
+      - GF_SECURITY_ADMIN_PASSWORD=grafana
+    volumes:
+      - ./grafana/provisioning:/etc/grafana/provisioning
+    networks:
```

```suggestion
      - GF_SECURITY_ADMIN_PASSWORD=grafana
    volumes:
      - ./docker/grafana/provisioning:/etc/grafana/provisioning
    networks:
```

Faltou corrigir o path aqui pro grafana subir já configurado. Também dá pra mover tudo que tá na pasta docker pra raiz. Tu que sabe!

---

Boa mestre, vou mover tudo para dentro de docker
Movi para fora para fazer um teste e esqueci de mover novamente